### PR TITLE
MPETS-525 - Investigate & Fix Differences in D2 - DSD Indicators

### PR DESCRIPTION
**Pull request by iteliolucas**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/EriDSDDataset.java [hunk 1]
```diff
@@ -112,25 +110,6 @@ public DataSetDefinition constructEriDSDDataset() {
...
             mappings),
         getChildrenColumn());
-    dsd.addColumn(
-        "D2BNP",
-        "Breastfeeding (exclude pregnant)",
-        EptsReportUtils.map(
-            eptsGeneralIndicator.getIndicator(
-                "D2BNP",
-                EptsReportUtils.map(
-                    eriDSDCohortQueries.getPatientsWhoAreBreastFeedingAndNotPregnant(), mappings)),
```
Comment (amosh101) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/EriDSDDataset.java:122]: These Cohort definitions should also be deleted.

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/EriDSDDataset.java [hunk 2]
```diff
@@ -112,25 +110,6 @@ public DataSetDefinition constructEriDSDDataset() {
...
-            mappings),
-        "");
-    dsd.addColumn(
-        "D2PNB",
-        "Pregnant (exclude breastfeeding)",
-        EptsReportUtils.map(
-            eptsGeneralIndicator.getIndicator(
-                "D2PNB",
-                EptsReportUtils.map(eriDSDCohortQueries.getPatientsWhoArePregnant(), mappings)),
```
Comment (amosh101) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/EriDSDDataset.java:131]: These Cohort definitions should also be deleted.

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/EriDSDDataset.java (hunk 1 above)
Comment (psmatsinhe) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/EriDSDDataset.java:122]: @amosh101 acho que ja tiraram nem?

**Comment by amosh101** [comment]
> @amosh101

just need to delete the queries now.

**Comment by amosh101** [comment]
> @amosh101  acho que ja tiraram nem?

Just need to delete the queries now.

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/EriDSDDataset.java (hunk 1 above)
Comment (iteliolucas) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/EriDSDDataset.java:122]: Done

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/EriDSDDataset.java (hunk 2 above)
Comment (iteliolucas) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/EriDSDDataset.java:131]: Done

**Pull request merged**